docs/tutorials/onnx/export_mxnet_to_onnx.md

```diff
@@ -45,11 +45,11 @@ Now, we have downloaded ResNet-18 symbol, params and synset file on the disk.
 
 Let us describe the MXNet's `export_model` API. 
 
-```python
+```
```

Comment:
+1, is default code-highlighting working just fine?

Reply:
When putting some code inside a ` ```python` block, this will automatically get converted into a cell in the notebook. Currently this tutorial cannot be executed when downloaded from the website, because the help instructions for this function are treated as python code. I also un-pythoned this code block to avoid printing twice the same content when downloading the notebook.